### ISSUE TITLE
Do not rename remotes

### ISSUE THREAD

**Issue by maxnoe** [cta-observatory/training-material#8]
In the tutorial, it is encouraged to do 

```
git remote add ctapipe https://github.com/cta-observatory/ctapipe.git
```

And to rename `origin` to `username-copy`.

While this seems like a good idea at first, there are conventions for forks. These conventions are used in git(hub) help and most tutorials. And, they are, you know, conventions.

I would recommend to stick to those conventions, which are:
- the default remote is called `origin` (which would be my fork of the project).
- the remote pointing to the mother project is called `upstream`

The line in the tutorial would thus become:

```
git remote add  upstream https://github.com/cta-observatory/ctapipe.git
```

And the renaming of the origin remote could be dropped. An Example for the use of this convention is the github fork help:  
https://help.github.com/articles/fork-a-repo/
and  
https://help.github.com/articles/syncing-a-fork/

**Comment by elehcim** [comment]
I agree, I'll change the presentation. This also created some confusion during the presentation. I usually use upstream and origin, I missed it. Thank you!
